test/pamela/HTN/symbol-ref-htn.main.htn.edn

```diff
@@ -0,0 +1,372 @@
+   {:args
+    [{:type :state-variable, :name state-var}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= state-var method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-65],
+  :incidence-set #{},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-38},
+ :hpt-39
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :field-ref, :names [field3]}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :field-ref, :names [field3]}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= field3 method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-66],
+  :incidence-set #{:hedge-65},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-39},
+ :hpt-40
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :field-ref, :names [field2 field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :field-ref, :names [field2 field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= field2.field1-1 method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-67],
+  :incidence-set #{:hedge-66},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-40},
+ :hpt-41
+ {:args
+  [{:type :equal,
+    :args
+    [{:type :method-arg-ref, :names [pclass-instance-arg field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}]}],
+  :argsmap
+  {"condition"
+   {:args
+    [{:type :method-arg-ref, :names [pclass-instance-arg field1-1]}
+     {:type :method-arg-ref, :names [method-arg]}],
+    :type :equal}},
+  :display-args [(= pclass-instance-arg.field1-1 method-arg)],
+  :display-name "Tell",
+  :edges [:hedge-68],
+  :incidence-set #{:hedge-67},
+  :name "tell",
+  :plant-id "bsm1",
+  :type :htn-primitive-task,
+  :uid :hpt-41},
+ :hpt-42
+ {:args [42],
+  :argsmap {"arg" 42},
+  :display-args [42],
+  :display-name "Simple Method",
+  :edges [:hedge-69],
+  :incidence-set #{:hedge-68},
+  :name "simple-method",
+  :type :htn-primitive-task,
+  :uid :hpt-42},
+ :hpt-43
+ {:args [{:type :state-variable, :name state-var}],
+  :argsmap {"arg" {:name state-var, :type :state-variable}},
+  :display-args [state-var],
+  :display-name "Simple Method",
+  :edges [:hedge-70],
+  :incidence-set #{:hedge-69},
+  :name "simple-method",
+  :type :htn-primitive-task,
+  :uid :hpt-43},
+ :hpt-44
+ {:args [field3],
+  :argsmap {"arg" field3},
+  :display-args [field3],
+  :display-name "Simple Method",
+  :edges [:hedge-71],
+  :incidence-set #{:hedge-70},
+  :name "simple-method",
+  :type :htn-primitive-task,
+  :uid :hpt-44},
+ :hpt-45
+ {:args
+  [{:type :pclass-ctor,
+    :uid :pclass-3,
+    :pclass pclass1,
+    :args [43],
+    :fields
+    {field1-1 {:access :private, :observable false, :initial 43}},
+    :ancestry [[:pclass-1 field2]]}],
```

Comment:
Although I believe all of the necessary information is in this `:args` reference, the representation of this isn't obvious.  As a field reference, I would have expected something like my suggestion for `:hpt-44`.  

What is the representation that is used for 2-level field references?  
Can we use a similar representation for both 1-level and 2-level field references (e.g., in `:hpt-44` and `:hpt-45`?